### PR TITLE
create about us page

### PR DESCRIPTION
**Pull request by ItsMatDev**

### CODE REVIEW

**Review by Cyrildab — APPROVED** [review]
It's nice.

File: frontend/src/components/AboutUs/CardProfil/CardProfil.jsx
Comment (Yoann-Fortin) [inline frontend/src/components/AboutUs/CardProfil/CardProfil.jsx:1]: Style inline is forbidden

File: frontend/src/components/AboutUs/CardProfil/CardProfil.scss
Comment (Yoann-Fortin) [inline frontend/src/components/AboutUs/CardProfil/CardProfil.scss:1]: How can you have 3 primary colors?

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: frontend/src/components/AboutUs/CardProfil/CardProfil.jsx
Comment (ItsMatDev) [reply frontend/src/components/AboutUs/CardProfil/CardProfil.jsx:1]: done

File: frontend/src/components/AboutUs/CardProfil/CardProfil.scss
Comment (ItsMatDev) [reply frontend/src/components/AboutUs/CardProfil/CardProfil.scss:1]: done

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**